### PR TITLE
Project 3: Using Bootstrap - Milestone 3

### PR DESCRIPTION
**Pull request by Vagyasri**
### We have created the following sections in the website for the 3 screen sizes:

- **In the magazine**
- **Editor’s pick**
- **Newspaper main sections**
         1. _U.S._
         2. _World_
         3. _Business_
         4. _Tech & Science_
        5. _Culture_
        6. _Sports_
        7. _Expert forum_
- **Subscribe / Newsletter**

Feel free to review and help us improve. If you like the work don't forget to give us
image

### CODE REVIEW

**Review by MkrtichSargsyan — CHANGES_REQUESTED** [review]
## _Status:_ Changes Requested :recycle:

## Hello Guys

You did an awesome job :+1:. However, there are some changes that you need to make in order to get your project approved

Note that you have two more reviews on this project before you reach the review limit, so make all of the required changes before submitting for a review

  ### In the magazine and Editor's pick section

- [x] titles should use hx tag
title


  ### Subscribe / Newsletter section

  - [x] Each section title uses a hx tag instead of div
  - [x] Newsletter section has an input with type email

  - [x] kindly fix this text overlapping (700 px)
  
overlap



  ### Main newspaper sections (US, World, etc.) section 

  - [x] Each title uses a hx tag and have a link

  <hr>

   dont use ```!important``` in css

## OPTIONAL

-[ ] You can use  ```object-fit: cover;``` for images in order to position them better

- [ ] Add h1 tag to the page

  theoretically and logically wrong to have subheadlines ```<h2>``` without a main headline ```<h1>```,   but it won't have a negative effect.

- [ ] Kindly use more bootstrap classes
    for example 
    > .subscribe-text1 {
    >    width: 100%;
    >  }
    you can use ```class="w-100"```

Please, make the REQUIRED changes and submit your request again! :grinning:
Good luck. 

> ps: . If you have any questions about my review or need help, you can message me on slack ( @mkrtichsargsyan )

File: style.css [hunk 1]
```diff
@@ -283,21 +287,37 @@ li {
...
-  .nav-upper {
-    display: none;
-  }
+.us:hover {
+  color: var(--custom-primary-color);
 }
 
+.indiana {
+  text-decoration: none;
+}
```
Comment (MkrtichSargsyan) [inline style.css:309]: You can use class="text-decoration-none" from bootstrap

File: style.css [hunk 2]
```diff
@@ -414,14 +438,174 @@ li {
...
+
+.signup-details {
+  width: 50%;
+  padding-left: 15px;
+}
+
+.subscribe-btn-new.signup {
+  width: 100%;
+}
```
Comment (MkrtichSargsyan) [inline style.css:562]: class="w-100" from bootstrap

File: style.css [hunk 3]
```diff
@@ -414,14 +438,174 @@ li {
...
+  }
+
+  .subscribe-btn-new {
+    background-color: var(--custom-primary-color);
+    width: 40%;
+  }
+
+  .date-container {
+    display: none;
+  }
```
Comment (MkrtichSargsyan) [inline style.css:587]: class="d-none" from bootstrap

File: style.css [hunk 4]
```diff
@@ -414,14 +438,174 @@ li {
...
+
+  .navlink-down {
+    display: none;
+  }
+
+  .nav-upper {
+    display: none;
+  }
```
Comment (MkrtichSargsyan) [inline style.css:595]: class="d-none" from bootstrap

File: style.css [hunk 5]
```diff
@@ -117,6 +122,7 @@ li {
   white-space: nowrap;
   line-height: 24px;
   text-decoration: none;
+  transition: var(--custom-transition);
```
Comment (Dipeshtwis) [inline style.css:125]: It's good to see that you are using variables in CSS.

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @Vagyasri  &  @barackm

Good job on implementing all the suggested changes. Well done.

**You may close this Pull Request and merge the changes**

## Optional

- [ ] I will highly recommend applying all those optional as well that are mentioned in the previous review.
 - To use bootstrap classes wherever necessary.

- [ ] From 769 px to 991px. this is looking different. fix that before merge.

### Your clone 

Screenshot (1324)

### Original

Screenshot (1325)


**_Apply these changes if you want to make your project even more perfect_**

**Keep being awesome devs and good luck with your next milestone!**


Cheers and Happy coding!👏👏👏

Code review by **Dipesh Kumar**
(Feel free to contact me on slack by the handle of  **deep** if you have any additional questions).
[Dipesh Kumar](https://deeptwist.tk) | [Twitter](https://twitter.com/97deepeshkumar)

**Pull request merged**